app/components/SmartAccountDemo.tsx

```
@@ -0,0 +1,105 @@
+} from '@app/components/ui/card';
+import { useToast } from '@app/hooks/use-toast';
+import { useSmartAccount } from '@app/services/smart-account';
+
+export function SmartAccountDemo() {
+  const { toast } = useToast();
+  const [isLoading, setIsLoading] = useState(false);
+  const [accountAddress, setAccountAddress] = useState<string>();
+
+  const smartAccount = useSmartAccount({
+    mode: 'default',
+    alchemyApiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)